app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -50,7 +50,7 @@ def textual_resource_version
 
   def textual_num_cpu_cores
```

Comment:
Look like we may eventually want to rename these methods too.  I believe there are also some aggreagation methods that inherit the same bad names.